docs/quickstart.ipynb

```diff
@@ -415,41 +772,178 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 28,
+   "execution_count": 14,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
        "param                            X1\n",
-       "t value           7.568059291000728\n",
+       "t value           7.568059291000726\n",
        "Pr(>|t|)                        0.0\n",
        "bootstrap_type                   11\n",
        "inference             CRV(group_id)\n",
        "impose_null                    True\n",
        "dtype: object"
       ]
      },
-     "execution_count": 28,
+     "execution_count": 14,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "fit2 = pf.feols(fml=\"Y~ X1\", data=data, vcov={\"CRV1\": \"group_id\"})\n",
+    "fit2 = pf.feols(fml=\"Y ~ X1\", data=data, vcov={\"CRV1\": \"group_id\"})\n",
     "fit2.wildboottest(param=\"X1\", B=999)"
```

Comment:
Maybe we can add a citation for wildboottest / the wild cluster bootstrap? The relevant paper is [MacKinnon, Nielsen & Webb, Journal of Applied Econometrics (2023)](http://qed.econ.queensu.ca/pub/faculty/mackinnon/working-papers/qed_wp_1485.pdf),

Reply:
Done in https://github.com/s3alfisc/pyfixest/pull/436/commits/eadcbf892160111010b44a6e8642ca220d690322